packaging/rpm/init.d/grafana-server

```diff
@@ -1,155 +1,136 @@
-#! /usr/bin/env bash
-
+#!/bin/bash
+#
+# grafana-server        Startup script for Grafana
+#
 # chkconfig: 2345 80 05
-# description: Grafana web server & backend
+# description: Grafana web server and backend
 # processname: grafana
 # config: /etc/grafana/grafana.ini
 # pidfile: /var/run/grafana.pid
-
+#
 ### BEGIN INIT INFO
 # Provides:          grafana
 # Required-Start:    $all
 # Required-Stop:     $remote_fs $syslog
 # Default-Start:     2 3 4 5
 # Default-Stop:      0 1 6
-# Short-Description: Start grafana at boot time
+# Short-Description: start and stop grafana-server
 ### END INIT INFO
 
-#  tested on
-#  1. New lsb that define start-stop-daemon
-#  3. Centos with initscripts package installed
+# init.d / servicectl compatibility (openSUSE)
+if [ -f /etc/rc.status ]; then
+    . /etc/rc.status
+    rc_reset
+fi
 
-PATH=/bin:/usr/bin:/sbin:/usr/sbin
-NAME=grafana-server
-DESC="Grafana Server"
+# Source function library
+. /etc/rc.d/init.d/functions
 
+# Set the default values for grafana-server variables
+PATH=/bin:/usr/bin:/sbin:/usr/sbin
+PROG=grafana-server
 GRAFANA_USER=grafana
 GRAFANA_GROUP=grafana
 GRAFANA_HOME=/usr/share/grafana
 CONF_DIR=/etc/grafana
-WORK_DIR=$GRAFANA_HOME
 DATA_DIR=/var/lib/grafana
 PLUGINS_DIR=/var/lib/grafana/plugins
 LOG_DIR=/var/log/grafana
 CONF_FILE=$CONF_DIR/grafana.ini
 MAX_OPEN_FILES=10000
-PID_FILE=/var/run/$NAME.pid
-DAEMON=/usr/sbin/$NAME
 
-if [ `id -u` -ne 0 ]; then
-  echo "You need root privileges to run this script"
-  exit 4
-fi
+PID_FILE=/var/run/$PROG.pid
+PID_FILE_TIMEOUT=10
+DAEMON=/usr/sbin/$PROG
 
-if [ ! -x $DAEMON ]; then
-  echo "Program not installed or not executable"
-  exit 5
-fi
-
-#
-# init.d / servicectl compatibility (openSUSE)
-#
-if [ -f /etc/rc.status ]; then
-    . /etc/rc.status
```

Comment:
You seem to have removed this (openSUSE) compatability stuff, is this not needed anymore for openSUSE ?